cli/commands/project_sync.go

```diff
@@ -74,8 +75,23 @@ func (cmd *ProjectSync) Commands() []cli.Command {
 func (cmd *ProjectSync) RunStart(ctx *cli.Context) error {
 	config := NewProjectConfig()
 	volumeName := cmd.GetVolumeName(ctx, config)
-	cmd.out.Verbose.Printf("Starting sync with volume: %s", volumeName)
 
+	switch platform := runtime.GOOS; platform {
+	case "darwin":
+		cmd.out.Verbose.Printf("Starting sync with volume: %s", volumeName)
+		cmd.StartUnisonSync(ctx, volumeName, config)
+	case "linux":
+		cmd.out.Verbose.Printf("Setting up local volume: %s", volumeName)
+		cmd.SetupBindVolume(volumeName)
+	default:
+		cmd.out.Verbose.Printf("Volume Sync not supported on: %s", platform)
```

Comment:
Does unison not work on Windows?

Reply:
Don't know, we have not tried it yet, but at a minimum the binary would likely be something like `unison.exe` so our current command would not be supported. I think windows also does cli options differently.

Reply:
Ah, so currently we are in aggressive conflict with Windows. Perhaps in the recent change to generator-outrigger-drupal, I should have it use the "devcloud" options for non-Darwin, instead of for linux specifically.

Reply:
I'm confused what hole are we leaving Windows users in? Is it "Add Unison support on Windows" or is it "Treat Windows like Linux"?

Reply:
Windows support would be "Treat Windows like Mac"

Reply:
Oh, I see what you mean.... let's default to what mac does and lets just use Linux as the single case.